src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3375,11 +3375,15 @@ private BoundNode BindConstructorBody(ConstructorDeclarationSyntax constructor,
             Debug.Assert(bodyBinder != null);
 
             if (constructor.Initializer?.IsKind(SyntaxKind.ThisConstructorInitializer) != true &&
-                ContainingType.GetMembersUnordered().OfType<SynthesizedRecordConstructor>().Any() &&
-                !SynthesizedRecordCopyCtor.IsCopyConstructor(this.ContainingMember()))
+                ContainingType.GetMembersUnordered().OfType<SynthesizedRecordConstructor>().Any())
```

Comment:
What is this change and why is it in this PR? Is something up with the merges in it?

Reply:
Soethign is broken with the merges.  this cahnge doesn't touch the compiler.  Sorry.  Will address.